lib/components/Datepicker/Datepicker.tsx

```diff
@@ -0,0 +1,106 @@
+import { DatePicker as ArkDatePicker, Portal, UseDatePickerContext } from '@ark-ui/react';
+
+interface HeaderProps {
+  datePicker: UseDatePickerContext;
+  targetView: UseDatePickerContext['view'];
+}
+const Header = ({ datePicker, targetView }: HeaderProps) => (
+  <ArkDatePicker.ViewControl className="mb-4 flex space-x-11">
+    <ArkDatePicker.PrevTrigger className="material-symbols-outlined">arrow_back</ArkDatePicker.PrevTrigger>
+
+    <button type="button" onClick={() => datePicker.setView(targetView)}>
+      <ArkDatePicker.RangeText className="capitalize" />
+    </button>
+
+    <ArkDatePicker.NextTrigger className="material-symbols-outlined">arrow_forward</ArkDatePicker.NextTrigger>
+  </ArkDatePicker.ViewControl>
+);
+
+export interface DatepickerProps {
+  /** Label text for input */
+  label: string;
+  /** Initial value, the internal format is ['yyyy-mm-dd'] */
+  value?: string[];
+  /** Callback value for when the value has been changed. Triggers on blur and on calendar click */
+  onValueChange?: (value: ArkDatePicker.ValueChangeDetails) => void;
+}
+export const Datepicker = ({ value, label, onValueChange }: DatepickerProps) => {
+  const tableCellClasses = 'p-4 hover:text-accent hover:underline focus:left-auto focus:underline capitalize';
+
+  return (
+    <ArkDatePicker.Root onValueChange={onValueChange} value={value} locale="fi-FI" className="inline-block">
+      <ArkDatePicker.Label className="mb-4 inline-block align-top text-form-label text-primary-gray">
+        {label}
+      </ArkDatePicker.Label>
+      <ArkDatePicker.Control>
+        <div className="inline-flex">
+          <ArkDatePicker.Input placeholder="(pp.kk.vvvv)" />
```

Comment:
This placeholder should be localized.

Reply:
@ketsappi  Should I add `i18next` to DS or should the placeholder be passed via props?

Reply:
We discussed this in different place and we ended up with the new prop.